src/app/views/provisioning/confirm.controller.coffee

```diff
@@ -21,6 +21,16 @@
       else
         $state.go('dashboard.provisioning.additional_details', params, {reload: reload})
 
+  setCustomSchema = () ->
```

Comment:
We need to change this to something like  this:
```
    parsedSchema = JSON.parse(product.custom_schema)
    schema = parsedSchema.json_schema || parsedSchema
    vm.form = parsedSchema.asf_options || ["*"]
    schemaForm.jsonref(schema)
      .then((schema) -> schemaForm.jsonref(schema))
      .then((schema) -> schemaForm.jsonref(schema))
      .then((schema) ->
        vm.schema = schema
        )
```
because we need to jsonref the vm.schema, in case it is namespaced under json_schema (we also don't need vm.model here).

Reply:
@iseessel @paulstuartparker why do we we call `schemaForm.jsonref(schema)` twice?

Reply:
Thats how it originally was done; here is the comment:  @ouranos @paulstuartparker

```
  # The schema is contained in field vm.product.custom_schema
  #
  # jsonref is used to resolve $ref references
  # jsonref is not cyclic at this stage hence the need to make a
  # reasonable number of passes (2 below + 1 in the sf-schema directive)
  # to resolve cyclic references
```